### PR TITLE
[Feature] Update function command create <Class name> <param 1> <para…

### PR DESCRIPTION
**Pull request by diegovivas**
…m 2> <param

### CODE REVIEW

File: console.py [hunk 1]
```diff
@@ -49,6 +49,34 @@ def do_create(self, line):
...
+            print("** class name missing **")
+        else:
+            if args[0] in self.all_classes:
+                new_instance = eval(args[0] + "()")
+                new_instance.save()
+                print(new_instance.id)
+                number_args = len(args)
+                count = 1
+                while (number_args > 1):
+                    foo = args[count].split("=")
```
Comment (jvcortes) [inline console.py]: don't use foo as variable names

File: console.py [hunk 2]
```diff
@@ -49,6 +49,34 @@ def do_create(self, line):
...
+                    x = foo[1]
+                    for conv in (int, float):
+                        try:
+                            x = conv(x)
+                            break
+                        except:
```
Comment (jvcortes) [inline console.py]: use except to catch a specific type of exception

File: console.py [hunk 3]
```diff
@@ -49,6 +49,34 @@ def do_create(self, line):
...
+                            pass
+                        if type(x) == str:
```
Comment (jvcortes) [inline console.py]: use isinstance(x) instead of type(x) to do a type check

File: console.py (hunk 1 above)
Comment (diegovivas) [reply console.py]: breve

File: console.py (hunk 2 above)
Comment (diegovivas) [reply console.py]: brevas

**Pull request merged**